kpi/views.py

```diff
@@ -520,7 +525,33 @@ def preview(self, request, *args, **kwargs):
             return Response(response_data, template_name='preview_error.html')
 
 
-class AssetViewSet(viewsets.ModelViewSet):
+class AssetVersionViewSet(NestedViewSetMixin, viewsets.ModelViewSet):
+    model = AssetVersion
+    lookup_field = 'uid'
+    filter_backends = (
+            AssetOwnerFilterBackend,
```

Comment:
Just a note: there is a [`RelatedAssetPermissionsFilter`](https://github.com/kobotoolbox/kpi/blob/0b9767fe044edf1dc37eca7c3725dc7ad2675151/kpi/filters.py#L98) if it'd be useful for non-owning users who have access to the asset to be able to see the versions. That filter [is used for snapshots](https://github.com/kobotoolbox/kpi/blob/0b9767fe044edf1dc37eca7c3725dc7ad2675151/kpi/views.py#L489).